src/wowman/core.clj

```diff
@@ -1034,6 +1064,32 @@
     (when-not (empty? full-data)
       (import-addon-list-v2 full-data))))
 
+;;
+
+(defn -upgrade-nfo
+  [addon]
+  ;; todo: this only needs to be done once. it's a data migration
+  (info "upgrading nfo file of addon" (:dirname addon))
+  (let [install-dir (get-state :selected-addon-dir)
+        ;; TODO: remove this in 0.14.0. it's reasonable to assume nfo files will consistently have a :game-track by then
```

Comment:
rm comment. if we filter for invalid nfo-v2 files, there will be no need to remove

Reply:
comment is referring to game-track munging, not the upgrade process. not removing